.devcontainer/Dockerfile

```diff
@@ -12,22 +11,23 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
     apt-get -y install --no-install-recommends apt-utils dialog && \
     apt-get -y install --no-install-recommends apt-transport-https ca-certificates software-properties-common libssl-dev libffi-dev \
         dotnet-sdk-3.1 build-essential gnupg-agent dnsutils httpie bash-completion curl wget git unzip jq lsb-release procps gnupg2 && \
+    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 0xB1998361219BD9C9 && \
+    curl -O https://cdn.azul.com/zulu/bin/zulu-repo_1.0.0-2_all.deb && \
+    apt-get install ./zulu-repo_1.0.0-2_all.deb && \
+    apt-get update && \
+    apt-get -y install zulu11-jdk && \
     apt-get -y upgrade
 
 COPY .devcontainer/library-scripts /tmp/library-scripts/
 
 # [Option] Install Maven
 ARG INSTALL_MAVEN="true"
 ARG MAVEN_VERSION="3.6.3"
-RUN if [ "${INSTALL_MAVEN}" = "true" ]; then su vscode -c "source /usr/local/sdkman/bin/sdkman-init.sh && sdk install maven \"${MAVEN_VERSION}\""; fi
-
-# [Option] Install Node.js
-ARG  INSTALL_NODE="false"
-ARG  NODE_VERSION="none"
-ENV  NVM_DIR=/usr/local/share/nvm
-ENV  NVM_SYMLINK_CURRENT=true \
-     PATH=${NVM_DIR}/current/bin:${PATH}
-RUN  if [ "$INSTALL_NODE" = "true" ]; then bash /tmp/library-scripts/node-debian.sh "${NVM_DIR}" "${NODE_VERSION}" "${USERNAME}"; fi
+RUN if [ "${INSTALL_MAVEN}" = "true" ]; then wget http://www-us.apache.org/dist/maven/maven-3/${MAVEN_VERSION}/binaries/apache-maven-${MAVEN_VERSION}-bin.tar.gz && \
+    tar -xf apache-maven-${MAVEN_VERSION}-bin.tar.gz && \
+    mv apache-maven-${MAVEN_VERSION}/ apache-maven/; fi
```

Comment:
not a lot of commands here but still would be better to move the commands to a `maven-debian.sh` script or something. each of the script files install the prerequisites like curl, ssl and other apt packages so we can't assume that wget will be installed when we get here. You can take a look at https://github.com/microsoft/vscode-dev-containers/blob/main/script-library/maven-debian.sh file from the upstream vscode-dev-containers repo and modify that to fit this installation